Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.20.2-alpine3.17 AS binarybuilder
+RUN apk --no-cache --no-progress add --virtual build-deps gcc musl-dev
+
+# Install Task
+RUN wget --quiet https://github.com/go-task/task/releases/download/v3.12.0/task_linux_amd64.tar.gz -O task_linux_amd64.tar.gz \
+  && sh -c 'echo "803d3c1752da31486cbfb4ddf7d8ba5e0fa8c8ebba8acf227a9cd76ff9901343  task_linux_amd64.tar.gz" | sha256sum -c' \
+  && tar -xzf task_linux_amd64.tar.gz \
+  && mv task /usr/local/bin/task
```

Comment:
We'd want multi-arch support so need to install task for different arch.

An example can be found in https://github.com/gogs/gogs/blob/f473643d29ff1d573e8f327f45ed0495aa2546a6/docker/build/install-task.sh#L5-L14

and the syntax is similar to https://github.com/asoul-sig/asouldocs/blob/4ac99cd225b870a687089593ddad97bb591f440a/Dockerfile#L23-L35